### PR TITLE
feat: Start conversation link

### PR DESCRIPTION
**Pull request by wadedvsa**
## Description

Add the ability to start a new conversation.
This feature needs to hide behind a feature toggle until it is needed.

Related issue: [VOL-4912](https://dvsa.atlassian.net/browse/VOL-4912)

## Before submitting (or marking as "ready for review")

- [ ] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
- [ ] Have you performed a self-review of the code
- [ ] Have you have added tests that prove the fix or feature is effective and working
- [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

File: module/Olcs/src/Controller/ConversationsController.php [hunk 1]
```diff
@@ -68,6 +69,14 @@ public function indexAction(): ViewModel
         return $view;
     }
 
+    public function newAction(): ViewModel
```
[the remark this answers is not in the document]
Comment (wadedvsa) [reply module/Olcs/src/Controller/ConversationsController.php:72]: The verb new just sounded right, you start a new conversation.

File: module/Olcs/config/module.config.php [hunk 2]
```diff
@@ -456,7 +456,16 @@
                         'action' => 'view',
                     ],
                 ]
-            ]
+            ],
+            'new' => [
```
[the remark this answers is not in the document]
Comment (wadedvsa) [reply module/Olcs/config/module.config.php:460]: Similar to the action, first verb that came to mind was new (conversation).

File: module/Olcs/src/Controller/ConversationsController.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (hobbyhacker0) [reply module/Olcs/src/Controller/ConversationsController.php:72]: I agree it fits the user journey well, however we've already got a few instances of "create" and "add" and I'm wary of introducing a 3rd.
In internal, there's a existing convention for addAction, editAction, deleteAction and some verbs are used in other properties & methods.

[the remark this answers is not in the document]
Comment (wadedvsa) [reply module/Olcs/src/Controller/ConversationsController.php]: Changed.

**Review by hobbyhacker0 — APPROVED** [review]

**Pull request merged**